src/main/java/com/wavefront/sdk/jaxrs/client/ClientHeadersInjectTextMap.java

```diff
@@ -0,0 +1,32 @@
+package com.wavefront.sdk.jaxrs.client;
+
+import io.opentracing.propagation.TextMap;
+
+import java.util.Iterator;
+import java.util.Map;
+
+import javax.ws.rs.core.MultivaluedMap;
+
+/**
+ * Helper class used to add carrier data to HTTP headers.
+ */
+public class ClientHeadersInjectTextMap implements TextMap {
```

Comment:
Wish [this](https://github.com/opentracing/opentracing-java/blob/master/opentracing-api/src/main/java/io/opentracing/propagation/TextMapInjectAdapter.java) used generics. Then we wouldn't need this class.